packages/beacon-node/src/network/network.ts

```diff
@@ -350,4 +358,24 @@ export class Network implements INetwork {
       }
     }
   };
+
+  private async onLightclientFinalityUpdate(finalityUpdate: altair.LightClientFinalityUpdate): Promise<void> {
```

Comment:
make these methods like:
```ts
private onBlah = async (...): Promise<...> => {}
```
so that you don't have to do `.bind(this)` in the constructor